lib/oga/xml/element.rb

```diff
@@ -91,6 +93,8 @@ def get(name)
         found ? found.value : nil
       end
 
+      alias [] get
+
```

Comment:
Please use `alias_method` instead of just `alias` as `alias_method` is used in other places.